### PR TITLE
Solving API link not sown in Github page 

### PR DESCRIPTION
**Pull request by amjadnoor**
## Related Issue
#107 solved

## Description
Now in Github page API is sown as a link that user can click and able to visit API.
You can check out [here](https://amjadnoor.github.io/life/)
## Additional
If you merge it I will be highly appreciate your team. 
Thanks 😊

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,12 +8,12 @@ A database and API for life-fe
 
 | Data                        | URL                                                                  |
 | --------------------------- | -------------------------------------------------------------------- |
-| List of districts with Data | https://life-api.coronasafe.network/data/active_district_data.json   |
-| Ambulance availability      | https://life-api.coronasafe.network/data/ambulance.json              |
-| Helpline Numbers            | https://life-api.coronasafe.network/data/helpline.json               |
-| Hospitals and Beds          | https://life-api.coronasafe.network/data/hospital_clinic_centre.json |
-| Medicine availability       | https://life-api.coronasafe.network/data/medicine.json               |
-| Oxygen availability         | https://life-api.coronasafe.network/data/oxygen.json                 |
+| List of districts with Data | <https://life-api.coronasafe.network/data/active_district_data.json>   |
```
Comment (bodhish) [inline README.md:11]: Please use markdown syntax.

**Review by bodhish — CHANGES_REQUESTED** [review]
Use markdown syntax 

[Docs](https://docs.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax#:~:text=You%20can%20create%20an%20inline,pages.github.com%2F)

**Comment by AmjadNoor** [comment]
Please checkout there [markdownguid.org](https://www.markdownguide.org/basic-syntax/#urls-and-email-addresses)
<kbd>To quickly turn a URL or email address into a link, enclose it in angle brackets.</kbd>

File: README.md (hunk 1 above)
Comment (AmjadNoor) [reply README.md:11]: It is not an inline link.

**Comment by bodhish** [comment]
@AmjadNoor Github uses its own flavor of markdown called as [gfm](https://github.github.com/gfm/)

It would be better if we use the [GFM syntax](https://github.github.com/gfm/#link-reference-definitions)

**Comment by AmjadNoor** [comment]
> @AmjadNoor Github uses its own flavor of markdown called as [gfm](https://github.github.com/gfm/)
> 
> It would be better if we use the [GFM syntax](https://github.github.com/gfm/#link-reference-definitions)

@bodhish  I know that GitHub uses it own markdown flavor.
Please check out this according to you [Github GFM docs](https://github.github.com/gfm/#example-602).
<kbd>Markdown 👇🏻👇🏻</kbd>
```Markdown
<https://github.com/coronasafe/life>
```
<kbd>HTML 👇🏻👇🏻</kbd>
```HTML
<p><a href="https://github.com/coronasafe/life">https://github.com/coronasafe/life</a></p>
```
In this situation both ***Markdown*** and **HTML** are same.

**Review by bodhish — APPROVED** [review]
Thanks for the changes 😅

**Pull request merged**